src/megaclient.cpp

```diff
@@ -14005,12 +14096,12 @@ bool MegaClient::syncup(LocalNode* l, dstime* nds, size_t& parentPending)
     return true;
 }
 
-bool MegaClient::syncup(LocalNode* l, dstime* nds)
-{
-    size_t numPending = 0;
-
-    return syncup(l, nds, numPending) && numPending == 0;
-}
+//bool MegaClient::syncup(LocalNode* l, dstime* nds, bool scanWholeSubtree)
```

Comment:
If this method has been discarded, we should remove it.
Otherwise, it's good to have a comment indicating why it's commented.